frontend/src/metabase/lib/settings.ts

```diff
@@ -179,10 +177,31 @@ class Settings {
     return this.get("hide-embed-branding?");
   }
 
-  ldapEnabled() {
+  isGoogleAuthConfigured() {
```

Comment:
Rename setting names to follow FE name convention and BE key names. e.g. `isEmailConfigured`